src/upsell_panda.js

```diff
@@ -48,19 +48,21 @@ function generatePromotionContent(promotion) {
   </div>`;
 }
 
-function renderPromotionInContainer(promotion) {
+function renderPromotionInContainer(promotion, fadeIn) {
   var bannerContainer = document.getElementsByClassName('__promotions_container')[0];
 
   var promotionHolder = document.createElement('span');
   promotionHolder.innerHTML = generatePromotionContent(promotion);
-  promotionHolder.classList = '__shown __promotion';
+  bannerContainer.classList = `__promotions_container ${fadeIn ? '__fade_in' : ''}`;
```

Comment:
In order for animation to work (it wasn't for me) we have to target container class.